src/viztracer/report_builder.py

```diff
@@ -67,10 +67,16 @@ def load_jsons(self) -> None:
                 self.jsons = [get_json(self.data)]
             elif isinstance(self.data, (list, tuple)):
                 self.jsons = []
+                corrupted_jsons = []
```

Comment:
Let's call it `self.invalid_json_paths` as this does not store json data and it might not be "corrupted" - could be not json at all.

We need this in the future because if there's no valid json files, we will raise an exception and the final message won't be printed.